docs/add-ons/kube-prometheus-stack.md

```diff
@@ -0,0 +1,28 @@
+# kube-prometheus-stack
+[kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)is a a collection of Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
+
```

Comment:
@alanty I think it's worth providing additional details about what is being deployed by this add-on with default config. e.g., Prometheus statefulset, kube-state-metrics, node-exporter, Grafana etc.

Reply:
updated a bit, we can link out to [templates](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack/templates) if we need to be more explicit. 

also added a note regarding the admin password similar to the changes in #746